Pearson-Advance/devstack
# PADV 291 - Getting the local environments ready

## Ticket

https://agile-jira.pearson.com/browse/PADV-291

## Description

This PR aims to create a branch for the Pearson-Advance/devstack that supports and configures the OpenedX Olive release.

## Type of Change

- Changing the sources from where the repos are cloned and the images pulled to fit our needs
- Adding scripts to configure ecommerce sites

## How to test

- Follow the steps for a standard devstack installation (clone the repo, make dev.clone, make dev.pull, make dev.provision)
- Once the provision is done check if the platform, ecom and the MFEs work correctly.
- Remember to update your hosts file with the following configuration:

```127.0.0.1       edx.devstack.lms
127.0.0.1       lms.pearson.localhost
127.0.0.1       lms.main.localhost

127.0.0.1       edx.devstack.ecommerce
127.0.0.1       ecommerce.pearson.localhost
127.0.0.1       ecommerce.main.localhost

127.0.0.1       edx.devstack.discovery
127.0.0.1       discovery.pearson.localhost
127.0.0.1       discovery.main.localhost
```

## Reviewers

The whole team is included so you can test it works on your local.
